Engine/Strings.resx

```diff
@@ -238,7 +238,7 @@
     <value>Key {0} in the settings is not a string.</value>
   </data>
   <data name="WrongKeyHashTable" xml:space="preserve">
-    <value>{0} is not a valid key in the settings hashtable. Valid keys are ExcludeRules, IncludeRules and Severity.</value>
+    <value>{0} is not a valid key in the settings hashtable. Valid keys are Severity, IncludeRules, ExcludeRules, Rules, CustomRulePath, RecurseCustomRulePath, and IncludeDefaultRules.</value>
```

Comment:
Looks good but I will re-order it to be alphabetically ordered if you agree with it (I did not think of this before)

Reply:
That makes sense (I agree). And thank you.